server/cluster.go

```diff
@@ -557,6 +557,25 @@ func (c *RaftCluster) checkStores() {
 	}
 }
 
+// RemoveTombStoneRecords remove the tombStone Records.
+func (c *RaftCluster) RemoveTombStoneRecords() error {
+	c.RLock()
+	defer c.RUnlock()
+
+	cluster := c.cachedCluster
+
+	for _, store := range cluster.GetStores() {
+		if store.IsTombstone() {
+			// the store has already been tombstone
+			err := cluster.deleteStore(store)
+			if err != nil {
+				return err
+			}
+		}
```

Comment:
Better to add a log if delete successfully.

Reply:
We want to know the deleted store through the log, so we also need to print the log if it is successfully deleted.

Reply:
Got it. :ok_hand: